src/mono/sample/iOS/Program.csproj

```
@@ -4,140 +4,57 @@
     <OutputPath>bin</OutputPath>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <TargetOS Condition="'$(TargetOS)' == ''">ios</TargetOS>
-    <TargetOS Condition="'$(TargetsiOSSimulator)' == 'true'">iossimulator</TargetOS>
-    <DeployAndRun Condition="'$(DeployAndRun)' == ''">true</DeployAndRun>
-    <iOSLikeDedup Condition="'$(TargetOS)' == 'ios'">true</iOSLikeDedup>
     <RuntimeIdentifier>$(TargetOS)-$(TargetArchitecture)</RuntimeIdentifier>
     <DefineConstants Condition="'$(ArchiveTests)' == 'true'">$(DefineConstants);CI_TEST</DefineConstants>
     <AppName>HelloiOS</AppName>
+    <MainLibraryFileName>Program.dll</MainLibraryFileName>
+    <DevTeamProvisioning Condition="'$(DevTeamProvisioning)' == ''">-</DevTeamProvisioning>
+    <DeployAndRun Condition="'$(DeployAndRun)' == ''">true</DeployAndRun>
     <StripDebugSymbols Condition="'$(StripDebugSymbols)' == ''">false</StripDebugSymbols>
-  </PropertyGroup>
+    <UseMonoJustInterp Condition="'$(UseMonoJustInterp)' == '' and '$(MonoForceInterpreter)' == 'true'">true</UseMonoJustInterp>
 
-  <PropertyGroup>
     <PublishTrimmed>true</PublishTrimmed>
     <TrimMode>Link</TrimMode>
+    <Optimized Condition="'$(ConQfiguration)' == 'Release'">True</Optimized>
```

```suggestion
    <Optimized Condition="'$(Configuration)' == 'Release'">True</Optimized>
```

There's an obvious typo. Also, isn't the option itself named `Optimize`?

---

Thanks. Property `Optimized` in AppleAppBuilder produce optimized binaries and use `Release` config in Xcode.

https://github.com/dotnet/runtime/blob/375661674c6c2c9d7c5d9b8f1d95f67ba653ce39/src/mono/msbuild/apple/build/AppleBuild.targets#L238-L264